Cargo.lock

```diff
@@ -52,8 +52,7 @@ checksum = "81cdd64d312baedb58e21336b31bc043b77e01cc99033ce76ef539f78e965ebc"
 [[package]]
 name = "zcash_spec"
 version = "0.1.2"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "9cede95491c2191d3e278cab76e097a44b17fde8d6ca0d4e3a22cf4807b2d857"
+source = "git+https://github.com/daira/zcash_spec.git?rev=0f687c5a12d12ae337ed8f85e65c5983bba114b2#0f687c5a12d12ae337ed8f85e65c5983bba114b2"
```

Comment:
I will rebase this when `zcash_spec 0.1.3` is released. (It appears not to have been tagged yet.)